recipes/geometric/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "geomeTRIC" %}
+{% set version = "0.8.8" %}
+{% set sha256 = "45ebea62a43045acd71d41516d732cb76bf65d760571a30454d4ff252dc1e10a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/leeping/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - geometric-optimize = geometric.optimize:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - {{ pin_compatible('numpy', min_pin='1.11') }}
```

Comment:
I think it's `lower_bound='1.11'`. `min_pin`/`max_pin` expect `x.x` format. [Example.](https://github.com/conda-forge/pyarrow-feedstock/blob/master/recipe/meta.yaml#L38) I'm still not sure [from docs](https://conda.io/docs/user-guide/tasks/build-packages/variants.html#extra-jinja2) if pin_compatible doesn't _require_ its target to be in `reqs/host`, but I'm curious to see the test.

You can change `reqs/build` to `reqs/host`, btw.